package.json

```diff
@@ -23,9 +23,10 @@
     "url": "git://github.com/facebook/regenerator.git"
   },
   "main": "main.js",
-  "bin": "bin/regenerator",
+  "bin": "bin/regenerator.js",
```

Comment:
ive changed it so this file could also be 'caught' by the `lint-staged`'s `*.js` rule